modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -470,10 +470,17 @@ const remove = (ed: Editor, name: string, vars?: FormatVars, node?: Node | Range
   // as there maybe nested bookmarks
   const isRemoveBookmarkNode = (node: Node) => Bookmarks.isBookmarkNode(node) && NodeType.isElement(node) && (node.id === '_start' || node.id === '_end');
 
+  const hasBlockChildren = (elm: Node) => Arr.exists(Arr.from(elm.childNodes), dom.isBlock);
```

Comment:
Since this is defined multiple times maybe move it to FormatUtils.ts also I think you might not need the Arr.from I think it accepts array like but haven't double checked.

Reply:
updated